packages/avatars/src/styled/utility.ts

```diff
@@ -6,16 +6,26 @@
  */
 
 import { getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { ThemeProps, DefaultTheme } from 'styled-components';
+import { math } from 'polished';
 
-import { STATUS } from '../types';
+import { STATUS, SIZE } from '../types';
 
+export const TRANSITION_DURATION = 0.25;
+
+const [xxs, xs, s, m, l] = SIZE;
 const [active, available, away, transfers, offline] = ['active', ...STATUS];
```

Comment:
These also feels brittle (and a bit too slick). Garden tends to get leverage out of TS by simply re-using the explicit union'd string values throughout.

Reply:
Good point about leveraging TS, updated in [37cf3bd](https://github.com/zendeskgarden/react-components/pull/1410/commits/37cf3bdeb0131ecf1f0a2da4ae655082a682133b).